zebra-chain/src/transaction/serialize.rs

```diff
@@ -342,3 +343,119 @@ where
         T::zcash_serialize(self, writer)
     }
 }
+
+/// A Tx Input must have an Outpoint (32 byte hash + 4 byte index), a 4 byte sequence number,
+/// and a signature script, which always takes a min of 1 byte (for a length 0 script)
+const MIN_TRANSPARENT_INPUT_SIZE: u64 = 32 + 4 + 4 + 1;
+/// A Transparent output has an 8 byte value and script which takes a min of 1 byte
+const MIN_TRANSPARENT_OUTPUT_SIZE: u64 = 8 + 1;
+// All txs must have at least one input and a 4 byte locktime
+const MIN_TRANSPARENT_TX_SIZE: u64 = MIN_TRANSPARENT_INPUT_SIZE + 4;
```

Comment:
The consensus rules in Zcash are slightly different, each transparent transaction must have an input *and* an output.

<img width="840" alt="Screen Shot 2021-03-18 at 08 41 45" src="https://user-images.githubusercontent.com/8951843/111724751-8791cb80-88b1-11eb-800e-66e1cc181de4.png">